isb_lib/core.py

```diff
@@ -227,9 +241,7 @@ def handle_produced_by_fields(coreMetadata: typing.Dict, doc: typing.Dict):
             if _shouldAddMetadataValueToSolrDoc(
                 location, "latitude"
             ) and _shouldAddMetadataValueToSolrDoc(location, "longitude"):
-                doc[
-                    "producedBy_samplingSite_location_latlon"
-                ] = f"{location['latitude']},{location['longitude']}"
+                lat_lon_to_solr(doc, location['latitude'], location['longitude'])
```

Comment:
This is really all it ended up being -- quite small since @datadavev did all the research and shapely does all the hard work.